Nikita_Konkov_Application/Selenium_Basics/Selenium_Basics/EpamHowWeDoItOurWorkNavigationTest.cs

```diff
@@ -0,0 +1,43 @@
+﻿using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Selenium_Basics
+{
+    [TestFixture]
+    public class EpamHowWeDoItOurWorkNavigationTest
+    {
+        private IWebDriver webDriver = new ChromeDriver();
+        private string _epamHowWeDoItLink;
+        private string _epamOurWorkLink;
+
+        [SetUp]
+        public void Setup()
+        {
```

Comment:
driver was not initialized in setup. See Clarification 5

Reply:
I cant do this in the "set up" because mistake appears.  Or you mean that that I should white command for opening browser.

Reply:
You can use property as a variable for the whole class. And initialize this property in setup. And you have initialized browser already for the whole class